articles/azure-resource-manager/bicep/bicep-config.md

```diff
@@ -36,30 +36,20 @@ The [Bicep linter](linter.md) checks Bicep files for syntax errors and best prac
 
 ## Enable experimental features
 
-You can enable preview features by adding:
+You can enable experimental features by adding the following section to your `bicepconfig.json` file.
+
+Here is an example of enabling features 'compileTimeImports' and 'userDefinedFunctions`. 
 
 ```json
 {
   "experimentalFeaturesEnabled": {
-    "userDefinedTypes": true,
-    "extensibility": true
+    "compileTimeImports": true,
+    "userDefinedFunctions": true
   }
 }
 ```
 
-> [!WARNING]
-> To utilize the experimental features, it's necessary to have the latest version of [Azure CLI](./install.md#azure-cli).
```

Comment:
I couldn't think of a reason why this was necessary.

Reply:
I don't recall who made the request.  It was added on March 3, 2023.